pkg/apis/ceph.rook.io/v1/zz_generated.deepcopy.go

```diff
@@ -1186,6 +1186,11 @@ func (in *CephFilesystemSubVolumeGroupList) DeepCopyObject() runtime.Object {
 func (in *CephFilesystemSubVolumeGroupSpec) DeepCopyInto(out *CephFilesystemSubVolumeGroupSpec) {
 	*out = *in
 	in.Pinning.DeepCopyInto(&out.Pinning)
+	if in.Quota != nil {
```

Comment:
These changes under `pkg/apis` actually are from master, not needed for your changes to `IsHost()`. I'd suggest just add these generated files in a separate commit in this PR. It seems our CI action is not working to catch that this generated code is out of date.